post-policy.go

```diff
@@ -24,8 +24,11 @@ import (
 	"time"
 )
 
-// expirationDateFormat date format for expiration key in json policy.
-const expirationDateFormat = "2006-01-02T15:04:05.999Z"
+const (
+	// expirationDateFormat date format for expiration key in json policy.
+	expirationDateFormat = "2006-01-02T15:04:05.999Z"
```

Comment:
Just a query, not a comment. Is expirationDateFormat similar to time.RFC3339Nano/time.RFC3339 constants.

Reply:
@BigUstad  yes they are RFC 3339 compliant dates.